docs/sources/auth/planning/index.md

```diff
@@ -0,0 +1,41 @@
+# Plan IAM integration strategy
+
+The following documentation is meant to shed light on the different authorization and authentications strategies available in Grafana. By doing preparation before implementation, the user will be able to decide which integration strategy suits best for their needs.
+
+## Where are my users?
+
+The first thing to consider is who are my users? Are the user exclusively within my organization or will they be outside of my organization?
+
+If the users are within my organization, this means that Grafana might be able to integrate with those users by providing connection with to the corresponding Identity Provider.
+
+If the users are outside of my organization, this means that Grafana needs to provide anonymous access, which is not enabled by default.
+
+### 🚧 How are my users organized?
```

Comment:
I'm guessing this would be a table of contents (`{{< section >}}`) and each of these sections would have an individual page, right?

I think it's useful to have it in this format for now, so we can see what remains to be done, but just wanted to check what your vision was :)

Reply:
Sorry for replying late to this comment. Yes! The idea of this section is to introduce the user to the different options. However, a more in-detail page will talk about specific Teams/Orgs/etc and used a reference in this section.